task/06-conditions-n-loops-tasks.js

```diff
@@ -81,8 +98,13 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,1,1   =>  false
  *   10,10,10 =>  true
  */
-function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+function isTriangle(a,b,c) 
+{
+    return ((Math.pow(a, 2) + Math.pow(b, 2) === Math.pow(c, 2)) ||
+        (Math.pow(b, 2) + Math.pow(c, 2) === Math.pow(a, 2)) ||
+        (Math.pow(c, 2) + Math.pow(a, 2) === Math.pow(b, 2)) ||
+        (a === b && a === c));
+    
```

Comment:
Cool, but you can do it in 1 row. Please, try to improve your code. Thank you!

Reply:
I thought is the string is big it is better to split it.

Reply:
No, I mean you can do it in another way and it will be much shorter

Reply:
You don't need Math.pow() here :)

Reply:
done